src/main/java/edu/harvard/iq/dataverse/api/RequestBodyException.java

```diff
@@ -0,0 +1,17 @@
+package edu.harvard.iq.dataverse.api;
+
+/**
+ *
+ * @author Jing Ma
+ */
+public class RequestBodyException extends Exception {
```

Comment:
These are fairly generic exceptions which would only be used for multilicense right now - I wonder if there's a way to reuse the existing exception handling patterns? (At some point, I'll grab the branch and look at the call trees - just reading the code in github right now.)

Reply:
Alternatively, we could try to reuse exceptions from the Java standard library. I think `RequestBodyException` could reasonably be replaced with an`IllegalArgumentException`. The other custom exception `ConflictException` could be a `IllegalStateException`, algthough a conflict is of course conceptually a special case of an illegal state.